<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Details</class>
 <widget class="QDialog" name="Details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Détails de l'employé</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>701</width>
     <height>311</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="lblDprenom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Prénom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrenom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDnom">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNom">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblphone">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Natel :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputphone">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sauver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>350</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Details</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
